proto/skaffold.proto

```diff
@@ -71,6 +82,12 @@ message BuildState {
     StatusCode statusCode = 3;
 }
 
+// `TestState` describes the status of the current test
+message TestState {
+    string status = 1;
+    StatusCode statusCode = 2;
```

Comment:
There is no code defines for Test failure. similar to `TEST_SUCCESS`  there needs to be failure codes.